README.md

```diff
@@ -1,6 +1,6 @@
 # Monix.io
 
-The homepage and the documentation website for the Monix project.
+The homepage and the documentation for the Monix project website.
```

Comment:
To me, it sounds like it talks about the documentation for the website, not the project.
Although I'm not a native speaker so I can trust you on that. :) (or we can just remove this word)